de4a-connector/src/main/webapp/WEB-INF/view/index.jsp

```diff
@@ -1,12 +1,81 @@
+<spring:eval expression="@environment.getProperty('de4a.kafka.url')" var="kafkaUrl" />
+<spring:eval expression="@environment.getProperty('smp.endpoint')" var="smpEndpoint" />
+<spring:eval expression="@environment.getProperty('idk.endpoint')" var="idkEndpoint" />
+<spring:eval expression="@environment.getProperty('phase4.send.toparty.id.type')" var="phase4SendToParty" />
+<spring:eval expression="@environment.getProperty('phase4.send.fromparty.id.type')" var="phase4SendFromParty" />
+<spring:eval expression="@environment.getProperty('toop.mem.implementation')" var="toopImpl" />
+<spring:eval expression="@environment.getProperty('as4.gateway.implementation.bean')" var="as4GatewayBean" />
+
+
 <!DOCTYPE HTML>
 <html xmlns:th="https://www.thymeleaf.org" lang="en">
 <head>
     <title>DE4A-Connector</title>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <style type="text/css">
+        body {
+            font-family: 'Helvetica';
+            color: #484848;
+        }
+        br {
+            content: "";
+            height: 5px;
+            display: block;
+        }
+        h4 {
+            margin: 15px auto;
+        }
+        .param-value {
+            font-weight: normal;		    
+		    font-family: 'Consolas';
+		    background: #eceaea;
+		    padding: 2px 2px;
+		    border: 1px solid #ccc;
+		    font-size: 14px;
+		    border-radius: 2px;
+        }
+        .value-true {
+            color: #5aa20b;
+        }
+        .value-false {
+            color: #e04747;
+        }
+        .value {
+            color: #484848;
+        }
+        .param-name {
+            font-family: 'Consolas';
+            color: #6b6b6b;
+            font-size: 13px;
+        }
+    </style>
 </head>
 <body>
-	<h1>Connector is up and running...</h1>
-	<hr>
-	<h3>Check out the system API on: </h3><a href="../swagger-ui/">Connector Swagger API definition</a>
+    <h1 style="color: #21739a;"><span>DE4A</span> - Connector</h1>
+    <h3>Status: <span class="param-value value-true">Running</span></h3>
+    <br>
+    <h3 style="margin-bottom: 5px;">System Parameters</h3>
+    <hr style="width: 40%;margin-left: 0;">
+	<h4 class="param-name">ssl.context.enabled: <span class="param-value ${sslContextEnabled ? ' value-true' : ' value-false'}">${sslContextEnabled}</span></h4>
+	<h4 class="param-name">http.proxy.enabled: <span class="param-value ${httpProxyEnabled ? ' value-true' : ' value-false'}">${httpProxyEnabled}</span></h4><br>
+	<h4 class="param-name">de4a.kafka.enabled: <span class="param-value ${kafkaEnabled ? ' value-true' : ' value-false'}">${kafkaEnabled}</span></h4>
+	<h4 class="param-name">de4a.kafka.http.enabled: <span class="param-value ${kafkaHttpEnabled ? ' value-true' : ' value-false'}">${kafkaHttpEnabled}</span></h4>
+	<h4 class="param-name">de4a.kafka.url: <span class="param-value value">${empty kafkaUrl ? 'not-setted' : kafkaUrl}</span></h4>
```

Comment:
Sorry to bug you. It should read "not-set" - it's a special case in English Grammar

Reply:
No problem at all. Thanks!